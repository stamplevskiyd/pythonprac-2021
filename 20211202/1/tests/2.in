class C(metaclass=dump):
    def __init__(self, val):
        self.real = val
    
    def __str__(self):
        return str(self.real)
        
    def __add__(self, number):
        return 2 * self.real + number.real  
    

c = C(10)
print(c + 15.3)
