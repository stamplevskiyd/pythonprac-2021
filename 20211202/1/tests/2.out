__init__: (10,), {}
__add__: (15.3,), {}
35.3
